src/views.js

```diff
@@ -157,6 +157,8 @@ var PersonViewModal = Backbone.View.extend({
 
         this.template = require('../static/templates/personViewModal.html');
         this.render();
+
+        jQuery.fn.editable.defaults.mode = 'inline';
```

Comment:
This directive only needs to be set once for the page. This line can be removed.